### ISSUE TITLE
new term request- Latent Mycobacterial Infection

### ISSUE THREAD

**Issue by pshultes** [obophenotype/human-phenotype-ontology#2621]
1. Preferred term label - Latent Mycobacterial Infections

2. Synonyms- Latent TB Infection, LTBI

3. Textual definition- Mycobacteria present without suffering from the associated disease or immune response; patient cannot spread disease to others unless latent infection transitions to the associated disease

PMID:26017823

4. Suggested parent term: Recurrent Bacterial Infections

5. N/A

6. Nanoattribution- UCB:pshultes

**Comment by pnrobinson** [comment]
Our preference would be to conceptualize entities such as latent mycobacterial infection as a disease and then to use HPO to describe the phenotypic features of it. We are going to be releasing more resources for common disease phenotype annotations in 2018, and we could use this as a test case. If you like, please send me an email at peter.robinson@jax.org
